packages/node_modules/@webex/helper-image/test/unit/spec/index.js

```diff
@@ -22,13 +45,32 @@ describe('helper-image', () => {
         objectType: 'file'
       };
 
-      return file.fetch('/Portrait_7.jpg')
-        .then((f) => {
-          readExifData(sampleFile, f)
```

Comment:
we were not returning previously so any failure in the `.then` below would mark the test as passed